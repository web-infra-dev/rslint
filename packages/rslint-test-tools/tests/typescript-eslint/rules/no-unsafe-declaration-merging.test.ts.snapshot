exports[`no-unsafe-declaration-merging > invalid 1`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 7,
  "ruleCount": 1
}
`;
