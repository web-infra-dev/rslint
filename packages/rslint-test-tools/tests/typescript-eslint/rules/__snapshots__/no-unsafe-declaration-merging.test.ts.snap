// Rstest Snapshot v1

exports[`no-unsafe-declaration-merging > invalid 1`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 0,
}
`;
